layer-utils/CMakeLists.txt

```diff
@@ -15,13 +15,12 @@
 # limitations under the License.
 #
 
-file(GLOB FILES_SOURCE ./*.cpp)
-file(GLOB FILES_HEADER ./*.h)
```

Comment:
Note. Don't use GLOB to add source/header files. Makes debugging CMake issues harder and can cause bugs on dirty builds.